packages/features/users/components/UserTable/EditSheet/EditUserForm.tsx

```diff
@@ -157,6 +160,7 @@ export function EditForm({
         </div>
       </div>
       <div className="mt-6 flex flex-col space-y-3">
+        <TextField label={t("username")} {...form.register("username")} />
```

Comment:
Do we need to check if username is available or not here (like we do in other pages)?

Reply:
but only within the organisation